app/src/main/kotlin/com/wire/android/ui/WireActivityViewModel.kt

```diff
@@ -138,6 +141,18 @@ class WireActivityViewModel @Inject constructor(
                 .distinctUntilChanged()
                 .collect { _observeSyncFlowState.emit(it) }
         }
+        viewModelScope.launch(dispatchers.io()) {
+            observeIfAppUpdateRequired(BuildConfig.VERSION_CODE)
```

Comment:
In the ticket description it says that it should be done once every 24h, will this be done in the next PR? 🤔

Reply:
that is done on kalium side in this UseCase (PR that you've reviewed :) )
there it works like this: 
`Flow` that emits every 24 hours -> get all the configs that were checked more than 24 hours ago from the DB -> check it and update `lastCheckDate`.